lib/modules/platform/github/index.ts

```diff
@@ -1494,12 +1494,12 @@ export async function createPr({
       base,
       body,
       draft: draftPR,
+      maintainer_can_modify: !!GlobalConfig.get('prAllowMaintainerEdits'),
```

Comment:
this should be moved back to the if condition below, as it only works with forkToken.